nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/flow-designer/ui/canvas/items/connection/prioritizers/prioritizers.component.html

```diff
@@ -16,53 +16,64 @@
   -->
 
 <div class="prioritizers flex flex-col gap-y-4" cdkDropListGroup>
-    <div class="flex flex-col">
-        <div>
-            Available Prioritizers
-            <i
-                class="fa fa-info-circle"
-                nifiTooltip
-                [tooltipComponentType]="TextTip"
-                tooltipInputData="Prioritizers that are available to reprioritize FlowFiles in this processors work queue."></i>
-        </div>
-        <div
-            class="prioritizers-list border"
-            cdkDropList
-            cdkDropListSortingDisabled
-            [cdkDropListDisabled]="isDisabled"
-            [cdkDropListData]="availablePrioritizers"
-            (cdkDropListDropped)="dropAvailable($event)">
-            @for (item of availablePrioritizers; track item; let i = $index) {
-                <div
-                    class="prioritizer-draggable-item border-b surface-contrast m-1"
-                    cdkDrag
-                    cdkDragPreviewContainer="parent">
-                    <ng-container
-                        *ngTemplateOutlet="
-                            prioritizerItem;
-                            context: { $implicit: item, i: i, canClose: false }
-                        "></ng-container>
-                </div>
-            }
+    @if (!isDisabled) {
+        <div class="flex flex-col">
+            <div>
+                Available Prioritizers
+                <i
+                    class="fa fa-info-circle"
+                    nifiTooltip
+                    [tooltipComponentType]="TextTip"
+                    tooltipInputData="Prioritizers that are available to reprioritize FlowFiles in this processors work queue."></i>
+            </div>
+            <div
+                class="cdk-list border"
+                cdkDropList
+                cdkDropListSortingDisabled
+                [cdkDropListDisabled]="isDisabled"
```

Comment:
With this change, this `cdkDropList` is only shown when `!isDisabled` so we do not need to set `cdkDropListDisabled`.